views/users/new.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta name='viewport' content='widht=devide-width,initial-scale=1'>
+    <title>Chitter</title>
+    <link href='/styles.css' rel="stylesheet" type="text/css">
+  </head>
+
+  <body>
+    <div class='registration'>
+      <div class='reg-form'>
+        <form action='/users' method='post'>
+          <input type='email' name='email' placeholder='Email' />
+          <input type='password' name='password' placeholder='Password' />
+          <div>
+          <input type='submit' value='Register' id='register-button' />
+        </form>
+      </div>
+    </div>
+  </body>
+
+</html>
```

Comment:
For the entire challenge:

* Features
  * [X] Sign up
  * [X] Log in
  * [X] Log out
  * [X] Peeping
  * [X] Listing Peeps (not in reverse chronological order)
  * [X] Display time of peep

* Bonus Features
  * [ ] Replying to peeps
  * [X] CSS styling